hub/tools/tagger.css

```diff
@@ -0,0 +1,123 @@
+body {
+    font-family: 'adobe-clean';
+    padding: 20px;
+    line-height: 1.4;
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+#results {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 150px;
+}
+
+#search {
+    width: 100%;
+    border: none;
+    border-bottom: 1px solid darkgrey;
+    font-size: 24px;
+    outline: none;
+    padding: 5px 0;
+}
+
+.filter {
+    display: flex;
+    flex-direction: column;
+    padding: 0 40px;
+}
+
+#selection {
+    background-color: #dfe4ea;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    padding: 10px;
+    width: 100%;
+    display: none;
+}
+
+.notInViewport {
```

Comment:
Maybe I'm missing something, but why not just make it a `hidden` input since it shouldn't be visible?

Reply:
selected tags are comma separated and stored as strings in the input; this will enable copying the values to the clipboard. using input type=hidden or display:none will not work.